src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -40,9 +48,15 @@ public LineResponse searchLine(final Long lineId) {
     }
 
     @Transactional
-    public void updateLine(final LineUpdateRequest lineUpdateRequest) {
-        Line line = lineRepository.findById(lineUpdateRequest.getId()).orElseThrow(IllegalArgumentException::new);
-        line.update(lineUpdateRequest.getName(), lineUpdateRequest.getColor());
+    public void updateLine(final LineUpdateRequest request) {
+        boolean existLineName = lineRepository.existsByName(request.getName());
+        Line line = lineRepository.findById(request.getId()).orElseThrow(IllegalArgumentException::new);
```

Comment:
```suggestion
        Line line = lineRepository.findById(request.getId())
                .orElseThrow(IllegalArgumentException::new);
```
한 줄에 다 쓰기 보다 적절한 엔터로 의미를 구분지어주세요.
그래야 코드 파악하기 좋습니다.

Reply:
stream과 같이 체이닝이 여러번 되는 경우가 아니라, 
위의 case 처럼 간단한 optional의 경우는 한 줄에 써서 전체적인 코드의 라인수를 줄이는게 코드 파악하는데 더 좋다고 생각이 들어서 저렇게 하는편인것 같습니다.

현구님의 경우에는 체이닝의 경우에 method 별로 줄바꿈을 하시는걸 더 선호하시는 건가요?

Reply:
팀에서 어떻게 정하느냐에 따라 다르겠네요.